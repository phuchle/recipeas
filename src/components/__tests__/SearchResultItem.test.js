import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import SearchResultItem from '../SearchResultItem';

describe('SearchResultItem', () => {
  test('renders without crashing', () => {
    const component = shallow(<SearchResultItem />);
    const tree = toJson(component);

    expect(tree).toMatchSnapshot();
  });
});
