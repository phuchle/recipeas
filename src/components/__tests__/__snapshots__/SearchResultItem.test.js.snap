// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchResultItem renders without crashing 1`] = `
<ListGroupItem
  bsClass="list-group-item"
  listItem={false}
>
  <Link
    replace={false}
    to={
      Object {
        "pathname": "/ingredient-details",
        "state": Object {
          "dbNumber": undefined,
        },
      }
    }
  />
</ListGroupItem>
`;
